<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontPropertyWidget</class>
 <widget class="QWidget" name="FontPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout" stretch="1,0,0,0,0,0">
   <item>
    <widget class="QComboBox" name="FontFamily">
     <property name="toolTip">
      <string>Select font</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <property name="minimumContentsLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="FontSize">
     <property name="toolTip">
      <string>Set font size</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="pqStandardColorButton" name="FontColor">
     <property name="toolTip">
      <string>Set font color</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="Bold">
     <property name="toolTip">
      <string>Bold</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Components/Resources/pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqBold16.png</normaloff>:/pqWidgets/Icons/pqBold16.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="Italics">
     <property name="toolTip">
      <string>Italics</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Components/Resources/pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqItalics16.png</normaloff>:/pqWidgets/Icons/pqItalics16.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="Shadow">
     <property name="toolTip">
      <string>Shadow</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Components/Resources/pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqShadow16.png</normaloff>:/pqWidgets/Icons/pqShadow16.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqStandardColorButton</class>
   <extends>QToolButton</extends>
   <header>pqStandardColorButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
